engine/src/main/java/org/hibernate/search/bridge/impl/NullEncodingCapable.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.bridge.impl;
+
+/**
+ * Marker interface to indicate that a StringBridge or FieldBridge implementation
+ * does not need wrapping to handle encoding of null values.
+ */
+public interface NullEncodingCapable {
```

Comment:
I'd name it `NullEncodingCapable(Field)Bridge` to make apparent it's a contract for (field) bridges.